src/main/java/cam72cam/immersiverailroading/tile/TileMultiblock.java

```diff
@@ -84,6 +90,35 @@ public int extractEnergy(int maxExtract, boolean simulate) {
     	}
     };
     
+    FluidTank tank = new FluidTank(10000);
+    
+    private IFluidHandler fluid = new IFluidHandler() {
```

Comment:
Why define your own IFluidHandler?  FluidTank already implements IFluidHandler?

Reply:
Otherwise, there is no FluidHandler attached to a Tile and then the Fluid Pipes (of IE) wouldn't connect.